index.js

```diff
@@ -3,7 +3,18 @@ const EN_LIST = ['en-US', 'en-GB', 'en', 'en-AU'];
 
 // public
 
-module.exports.ARGUMENTS = ['--restrict-filenames', '--write-sub', '--write-auto-sub', '--max-downloads', '1', '-f', '[height <=? 720][format_id != source]', '-j'];
+module.exports.isPlaylist = (url) => {
+  return url.startsWith('https://www.youtube.com/playlist?list=');
+};
+
+module.exports.getArguments = (isPlaylist) => {
```

Comment:
breaking API for no reason? why not just have a separate property with the playlist args

Reply:
Oh tru